code/workspaces/infrastructure-api/src/controllers/stackController.js

```diff
@@ -168,9 +168,9 @@ const restartStackValidator = [
 const updateStackValidator = [
   checkExistsWithMsg('name'),
   checkExistsWithMsg('projectKey'),
-  ...USER_UPDATEABLE_FIELDS.map(field => body(field).optional()),
+  ...USER_UPDATEABLE_FIELDS.map(field => body(field).optional({ nullable: true })),
```

Comment:
Shouldn't this bug fix have generated a test?

Reply:
Yes, it really should have. Thanks for picking this up Andy!